### PR TITLE
feat: sentence transformer

### PR DESCRIPTION
**Pull request by slettner**

### CODE REVIEW

File: tests/integration/test_flow_integration.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            "c",
+        ),
+        (
+            pytest.lazy_fixture("docs_with_chunk_chunk_text"),
+            [["r", 0], ["c", 0], ["cc", 10]],
+            "cc",
+        ),
+    ],
+)
+def test_traversal_path(
```
Comment (JoanFM) [inline tests/integration/test_flow_integration.py:44]: this must be unit test

File: sentence_encoder.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            [doc for doc in flat_docs if doc is not None and doc.text is not None]
+        )
+        return _batch_generator(filtered_docs, batch_size)
+
+
+def _batch_generator(
+    data: DocumentArray, batch_size: int
+) -> Generator[DocumentArray, None, None]:
+    for i in range(0, len(data), batch_size):
+        yield data[i: min(i + batch_size, len(data))]
```
Comment (JoanFM) [inline sentence_encoder.py:58]: do not check min here, not needed

**Review by JoanFM — DISMISSED** [review]
single quotes please

File: sentence_encoder.py (hunk 2 above)
Comment (Yongxuanzhang) [reply sentence_encoder.py]: fixed

File: tests/integration/test_flow_integration.py (hunk 1 above)
Comment (Yongxuanzhang) [reply tests/integration/test_flow_integration.py]: done

**Review by Yongxuanzhang — APPROVED** [review]

**Pull request merged**